.travis.yml

```diff
@@ -3,25 +3,62 @@
 # All rights reserved.
 # END COPYRIGHT BLOCK
 
+language: java
+
 services:
   - docker
 
-env:
-  - FEDORA=27
-  - FEDORA=28
-
-install:
-  - docker pull registry.fedoraproject.org/fedora:$FEDORA
-  - docker run
-      --name=container
-      --detach
-      -i
-      -v $(pwd):/root/jss
-      registry.fedoraproject.org/fedora:$FEDORA
-  - docker exec container dnf install -y dnf-plugins-core gcc make rpm-build
-  - docker exec container dnf copr -y enable ${JSS_4_5_REPO:-@pki/10.6}
-  - docker exec container dnf builddep -y --spec /root/jss/jss.spec
-  - docker exec container /root/jss/build.sh --with-timestamp --with-commit-id rpm
-
-script:
-  - docker exec container rpm -Uvh /root/build/jss/RPMS/*
+stages:
+  - validate
+  - build_rpm
+
+jobs:
+  include:
+    # Validate style of scripts
+    - stage: validate
```

Comment:
How about moving this stage to existing scripts and keeping the matrix? -- The matrix is to avoid rewriting the same set of scripts :)